src/BikramSambat.ts

```diff
@@ -22,4 +22,10 @@ export default class BikramSambat {
     this.month = month
     this.day = day
   }
+  public getPreviousMonth():number{
+    return this.month - 1
+  }
+  public getNextMonth():number{
+    return this.month + 1
+  }
```

Comment:
Same here, we will need to account month 12.
you can calculate month like `const month = (this.month+1)%12`
then return the month data from [NepaliMonthsData](https://github.com/askbuddie/nepali-date-picker/blob/77537082d4b192929347856fbc2de443a2513845/src/data/nepali-months.ts#L1)

Reply:
so we should be returning this instead.
const month = (this.month + 1) %12
return month

Reply:
no no, this is just to get the month index, you should be returning the object like `{index: 12, en: 'Chaitra', np: 'चैत्र', ad: 'Mar/Apr' }` from [NepaliMonthsData](https://github.com/askbuddie/nepali-date-picker/blob/77537082d4b192929347856fbc2de443a2513845/src/data/nepali-months.ts#L1)
it should be like
```
public getNextMonth():Month{
    if(!this.month){
    return null
    }
    const month = (this.month+1)%12
    return NepaliMonthsData[month-1]
  }
```

same for getPreviousMonth

Reply:
solution implemented . Review please.